.changeset/common-cycles-obey.md

```diff
@@ -0,0 +1,11 @@
+---
+"@neo4j/cypher-builder": major
+---
+
+Remove support for patterns in size.
+
+_No longer supported_
+
+```js
+Cypher.size(new Cypher.Pattern(node));
```

Comment:
Only thing I can think of would be to add a snippet for "Here's what you should do instead"